README.md

```diff
@@ -28,24 +28,27 @@ System Administrators and DevOps who wish to send a notification now no longer n
 
 # Table of Contents
 <!--ts-->
-* [Supported Notifications](#supported-notifications)
-  * [Productivity Based Notifications](#productivity-based-notifications)
-  * [SMS Notifications](#sms-notifications)
-  * [Desktop Notifications](#desktop-notifications)
-  * [Email Notifications](#email-notifications)
-  * [Custom Notifications](#custom-notifications)
-* [Installation](#installation)
-* [Command Line Usage](#command-line-usage)
-  * [Configuration Files](#cli-configuration-files)
-  * [File Attachments](#cli-file-attachments)
-  * [Loading Custom Notifications/Hooks](#cli-loading-custom-notificationshooks)
-  * [Environment Variables](#cli-environment-variables)
-* [Developer API Usage](#developer-api-usage)
-  * [Configuration Files](#api-configuration-files)
-  * [File Attachments](#api-file-attachments)
-  * [Loading Custom Notifications/Hooks](#api-loading-custom-notificationshooks)
-* [Persistent Storage](#persistent-storage)
-* [More Supported Links and Documentation](#want-to-learn-more)
```

Comment:
Only thing I have left is to restore all of this, the only change made to the readme should be the 'dot' entry further down.

Otherwise I can't spot anything else. Great work. This is very close to a merge and is a full service. Very impressive 🚀